sdk/contentsafety/ai-content-safety-rest/review/ai-content-safety.api.md

```diff
@@ -0,0 +1,575 @@
+}
+
+// @public (undocumented)
+export interface DeleteTextBlocklistDefaultResponse extends HttpResponse {
+    // (undocumented)
+    body: ErrorResponse;
+    // (undocumented)
+    headers: RawHttpHeaders & DeleteTextBlocklistDefaultHeaders;
+    // (undocumented)
+    status: string;
+}
+
+// @public (undocumented)
+export type DeleteTextBlocklistParameters = RequestParameters;
+
+// @public
+export type GetArrayType<T> = T extends Array<infer TData> ? TData : never;
+
+// @public
+export type GetPage<TPage> = (pageLink: string, maxPageSize?: number) => Promise<{
+    page: TPage;
+    nextPageLink?: string;
+}>;
+
+// @public (undocumented)
+export interface GetTextBlocklist {
+    delete(options?: DeleteTextBlocklistParameters): StreamableMethod<DeleteTextBlocklist204Response | DeleteTextBlocklistDefaultResponse>;
+    get(options?: GetTextBlocklistParameters): StreamableMethod<GetTextBlocklist200Response | GetTextBlocklistDefaultResponse>;
+    patch(options: CreateOrUpdateTextBlocklistParameters): StreamableMethod<CreateOrUpdateTextBlocklist200Response | CreateOrUpdateTextBlocklist201Response | CreateOrUpdateTextBlocklistDefaultResponse>;
+}
+
+// @public
+export interface GetTextBlocklist200Response extends HttpResponse {
+    // (undocumented)
+    body: TextBlocklistOutput;
+    // (undocumented)
+    status: "200";
+}
+
+// @public (undocumented)
+export interface GetTextBlocklistDefaultHeaders {
+    "x-ms-error-code"?: string;
+}
+
+// @public (undocumented)
+export interface GetTextBlocklistDefaultResponse extends HttpResponse {
+    // (undocumented)
+    body: ErrorResponse;
+    // (undocumented)
+    headers: RawHttpHeaders & GetTextBlocklistDefaultHeaders;
+    // (undocumented)
+    status: string;
+}
+
+// @public (undocumented)
+export interface GetTextBlocklistItem {
+    get(options?: GetTextBlocklistItemParameters): StreamableMethod<GetTextBlocklistItem200Response | GetTextBlocklistItemDefaultResponse>;
+}
+
+// @public
+export interface GetTextBlocklistItem200Response extends HttpResponse {
+    // (undocumented)
+    body: TextBlockItemOutput;
+    // (undocumented)
+    status: "200";
+}
+
+// @public (undocumented)
+export interface GetTextBlocklistItemDefaultHeaders {
+    "x-ms-error-code"?: string;
+}
+
+// @public (undocumented)
+export interface GetTextBlocklistItemDefaultResponse extends HttpResponse {
+    // (undocumented)
+    body: ErrorResponse;
+    // (undocumented)
+    headers: RawHttpHeaders & GetTextBlocklistItemDefaultHeaders;
+    // (undocumented)
+    status: string;
+}
+
+// @public (undocumented)
+export type GetTextBlocklistItemParameters = RequestParameters;
+
+// @public (undocumented)
+export type GetTextBlocklistParameters = RequestParameters;
+
+// @public
+export interface ImageAnalyzeSeverityResultOutput {
+    category: string;
+    severity: number;
+}
+
+// @public
+interface ImageData_2 {
+    blobUrl?: string;
+    content?: string;
+}
+export { ImageData_2 as ImageData }
```

Comment:
@qiaozha it looks like this is happening here. Not sure why. IIRC when this happens the actual type that is exported will be ImageData not ImageData_2

Reply:
I wonder why this is the only model has a _2 suffix here.